apps/api/src/.env.test

```diff
@@ -84,7 +84,7 @@ NOVU_SMS_INTEGRATION_TOKEN=test
 NOVU_SMS_INTEGRATION_SENDER=1234567890
 
 IS_API_IDEMPOTENCY_ENABLED=true
-AUTO_CREATE_INDEXES=true
+MONGO_AUTO_CREATE_INDEXES=true
```

Comment:
@tatarco Can we change in all relevant env files? 

- Any other service
- local/production(keep it `false` in production)/example env files